Dockerfile

```diff
@@ -1,13 +1,28 @@
 # base image
-FROM node:12.2.0-alpine
+# Vite did not like the older version of Node that we'd specified, so I switched to the most recent LTS version
+FROM node:hydrogen-alpine
+
 
 # set working directory
 WORKDIR /app
 
 # install and cache app dependencies
 COPY package.json /app/package.json
 RUN npm install
-RUN npm install @vue/cli@3.7.0 -g
 
-# # start app
-CMD ["npm", "run", "serve"]
\ No newline at end of file
+# Without this line the page will not load
+COPY . . 
```

Comment:
That's weird as it works for me without this line. The bind mounts set in the docker-compose ought to cover this. Please can you share the error you got and the command you ran.

Reply:
And now it's working for me as well.  I can only assume that I misidentified the problem somewhere along the line and solved it some other way.  (I was getting the error "server reset during page load.")

Reply:
Fixed with [3fa681c](https://github.com/wikimedia/toolhunt-ui/pull/5/commits/3fa681c0eee04566eab18fb88bc2e2683f55c95e)